### PR TITLE
Add playbook to deploy HAProxy configurations

### PR DESCRIPTION
**Pull request by jhriv**

### CODE REVIEW

**Comment by jhriv** [comment]
See https://github.com/UCLALibrary/ansible_uclalib_configs/pull/208

File: uclalib_haproxy.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      file:
+        name: "{{ haproxy_config_dir }}/{{ item }}"
+        state: directory
+      become: true
+      loop:
+        "{{ haproxy_conf_directories.stdout_lines }}"
+
+    - name: Copy Files
+      copy:
+        src: "{{ haproxy_local_file_path }}/{{ item }}"
```
Comment (cachemeoutside) [inline uclalib_haproxy.yml:68]: I'm a little confused by this line, shouldn't the source be referencing the controller files path? I guess I was expecting:
```
"files/{{ haproxy_local_file_path }}/{{ item }}"
```

File: uclalib_haproxy.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        dest: "{{ haproxy_config_dir }}/{{ item | regex_replace ('\\.vault(|\\..*)$', '\\1' ) }}"
+      become: true
+      loop:
+        "{{ haproxy_conf_files.stdout_lines }}"
+
+    - name: Copy Config
+      copy:
+        src: "{{ haproxy_local_file_path }}/haproxy.cfg"
```
Comment (cachemeoutside) [inline uclalib_haproxy.yml:76]: same comment as above ^^

File: uclalib_haproxy.yml (hunk 1 above)
Comment (jhriv) [reply uclalib_haproxy.yml:68]: "files" is in the Ansible controller search path for the "copy" module

**Pull request merged**